utils/utils.go

```diff
@@ -0,0 +1,5 @@
+package utils
```

Comment:
Please move this to commons, I know a small patch + release is a hassle 😅  but we have this func in all repos